src/utils/translations.ts

```diff
@@ -1,10 +1,15 @@
 import type { Lang } from "./languages"
 import type { Direction } from "../types"
 
+import i18nConfigs from "../../i18n/config.json"
+
 export type TranslationKey = string
 
 export const isLangRightToLeft = (lang: Lang): boolean => {
-  return lang === "ar" || lang === "fa"
+  const langConfig = i18nConfigs.filter(
+    (language) => language.hrefLang === lang
```

Comment:
I think we want to use `language.code` here, not `hrefLang`

ie: Filipino we use the internal code `ph`, but the formal language code is `fil`... in `/utils/languages.ts` we use `ph`, not `fil` (which is where we declare the `Lang` type and it's allowed options)

```suggestion
    (language) => language.code === lang
```

Reply:
Ah, this makes sense. Great catch!